vendor/github.com/containers/common/pkg/config/default.go

```diff
@@ -37,6 +37,30 @@ const (
 )
 
 var (
+	DefaultMaskedPaths = []string{
+		"/proc/acpi",
+		"/proc/kcore",
+		"/proc/keys",
+		"/proc/latency_stats",
+		"/proc/sched_debug",
+		"/proc/scsi",
+		"/proc/timer_list",
+		"/proc/timer_stats",
+		"/sys/dev/block",
```

Comment:
This appears to allow the rest of `/sys/dev`, which we didn't before.

Reply:
Podman switched from /sys/dev to /sys/dev/block a while ago.  So I went with the Podman default.

Reply:
Is it correct for build-time as well as run-time?

Reply:
I would figure.  Are concerned about access to content in /sys/dev?  Note this is looser then buildah had before.

Reply:
It being looser is what prompted my question.  https://github.com/containers/podman/pull/6957 masked all of /sys/dev without being more specific about what under it mattered, https://github.com/containers/podman/pull/8408 narrowed the default to /sys/dev/block, and https://github.com/containers/podman/issues/12746 is asking to unmask the only part under it that we're still going to be masking.

Reply:
Yup, just going for consistency now.